apsuite/optics_analysis/coupling_correction.py

```diff
@@ -13,11 +13,13 @@ class CouplingCorr():
 
     CORR_STATUS = _namedtuple('CorrStatus', ['Fail', 'Sucess'])(0, 1)
 
-    def __init__(self, model, acc, dim='4d', skew_list=None):
+    def __init__(self, model, acc, dim='4d',
+                 skew_list=None, method='orbrespm'):
         """."""
         self.model = model
         self.acc = acc
         self.dim = dim
+        self.method = method
```

Comment:
I think it is nice to create a namedtuple for the possible methods:

`METHODS = _namedtuple('Methods', ['Additional', 'Proportional'])(0, 1)`

keep the method attribute as private object attribute (`self._method`) and create a property for it, just like it is done for the optics correction.

Reply:
done!